Route path for reset password included

Ade0211 opened the pull request:
1) I  included the reset password route  
  2) I installed materialize-css in order to implement my M.toast
3) i changed the homepage path from (/front) to (/)

Ade0211 committed:
Routes

1) I  included the reset password route    2) I installed materialize-css in order to implement my M.toast
3) i changed the homepage path from (/front) to (/)

ISTANBULBEKLE reviewed (approved):
Great job guys

Wolfjawan commented on src/components/ResetPassword/resetpassword.jsx:
```diff
@@ -12,8 +12,8 @@ export default function ResetPassword() {
-
   const PostData = () => {
     console.log("hi");
     if (re) {
```
How this if statement checks the email?

Ade0211 replied:
I have changed the pattern in which the email received will be validated. Thanks for bringing my notice to this

Wolfjawan commented on src/components/ResetPassword/resetpassword.jsx:
```diff
@@ -12,8 +12,8 @@ export default function ResetPassword() {
   const re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9] {1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-
   const PostData = () => {
     console.log("hi");
```
console log should be removed

Ade0211 committed:
Reset Email validation

1) requested changes on email validation done
2) debugger (Console.log) removed

Wolfjawan reviewed (approved):
LGTM 👍🏼

Denis-Andrei reviewed (approved):
Looks good
